### ISSUE TITLE
High contrast issues when interacting with tab stops requirements and instances in Edge

### ISSUE THREAD

**Issue by jalkire** [microsoft/accessibility-insights-web#5097]
## Describe the bug
When interacting with the tab stops requirements and instances sections with a Windows high contrast mode enabled and using Edge, coloring becomes inconsistent and potentially unreadable.

**To Reproduce**

1. Install the Insider or Canary extension (not the current prod, 2.29.0) on Edge
2. Enable High Contrast in the Windows settings. The issue is more or less obvious with different contrast themes; Dusk and Night sky are good options in Windows 11.
3. Navigate to Tab stops in FastPass
4. Mark requirements as passed or failed and otherwise interact with the page
5. Observe strange coloring

## Expected behavior
Colors should not change dramatically upon interaction

## Screenshots
![GIF showing the behavior described above](https://user-images.githubusercontent.com/4615491/150612984-534768b4-5375-44af-a939-42fe1501d0a3.gif)

## Context (please complete the following information)

- OS Name & Version: Windows 11 22000.434
- AI-Web Version & Environment: 2.30.0 insider
- Browser Version: Edge Version 97.0.1072.69 (Official build) (64-bit)

## Additional context
Found during release validation for 2.30.0

**Comment by jalkire** [comment]
Confirmed fixed in insider 2.30.1
